lib/extensions/require_nested.rb

```diff
@@ -15,6 +15,14 @@ def require_nested(name)
       Object.require_nested name
     end
   end
+
+  def require_nested_all(path)
+    path = "#{path[0...-3]}/*.rb" # i.e. 'mod/abc.rb' => 'mod/abc/*.rb'
+    Dir.glob(path).sort.each do |full_path|
+      name = File.basename(full_path)[0...-3]
```

Comment:
basename second parameter can remove the extension

File.basename(full_path, "*.rb")

Reply:
nice! done!
 (And should `be File.basename(full_path, ".rb")`) 😄